pageserver/src/pgdatadir_mapping.rs

```diff
@@ -519,14 +553,15 @@ impl Timeline {
             result.add_key(rel_dir_to_key(spcnode, dbnode));
 
             let mut rels: Vec<RelTag> = self
-                .list_rels(spcnode, dbnode, lsn)
+                .list_rels(spcnode, dbnode, lsn, ctx)
                 .await?
-                .into_iter()
+                .iter()
```

Comment:
NIT: Is there any reason to moving from `into_iter` to `iter` + `cloned`?

Reply:
No, just fallout from the extraction.